app/scripts/components/account/subscription-choose-plan.components.jsx

```diff
@@ -84,7 +84,8 @@ export default class SubscriptionChoosePlan extends React.Component {
 					label="Coupon code"
 					placeholder="ABC123"
 					value={this.state.couponValue}
-					handleOnChange={this.handleCouponChange} />
```

Comment:
It was the only `handleOnChange`, I find `onChange` a better name since it's fired when an event is triggered.

The standard `ref` works exactly as before, but when `children` is set and you're trying to access `inputValue`, it warns you to set the `ref` on the child instead.